config/constants/socket.js

```diff
@@ -0,0 +1,15 @@
+const socketConst = {
+  getPost: 'get_post',
```

Comment:
I prefer when constants in upper case, but is just a matter of taste.
socketConst.GET_POST  - like this.